docs/extensions/bridge/bridge.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Bridge
+---
+
+import {
+  DiscordButton,
+  DiscordButtons,
+  DiscordInteraction,
+  DiscordMessage,
+  DiscordMessages,
+} from "@discord-message-components/react";
+import "@discord-message-components/react/styles";
+
+## Concept
+
+Let's say that you want to make a command that is both a Slash Command and a Prefixed Command. Now, you could just copy and paste the code from the first command callback to the other and make some adjustments, but that's not very efficient.
+
+This is where the `ext.bridge` comes in. It allows you to use one callback to make both a Slash Command and a Prefixed Command.
+
+:::note
+Currently, `ext.bridge` has not been merged to the main branch yet and it's somewhat experimental. So in order to use it, you'll have to install the `py-cord` package from the `ext-compat` branch by running this:
+```
+pip install git+https://github.com/Pycord-Development/pycord.git@ext-compat
+```
+:::
+
+### Example Usage
+
+```python
+import discord
+from discord.ext import bridge
+
+bot = bridge.Bot(command_prefix="!")
+
+@bot.bridge_command()
+async def hello(ctx):
+  await ctx.respond("Hello!")
+
+bot.run("TOKEN")
+```
+
+<DiscordMessages>
```

Comment:
In ext.bridge, respond correlates to reply instead of send. Please reflect this in these examples.
![Screen Shot 2022-03-26 at 1 26 56 PM](https://user-images.githubusercontent.com/71356958/160252478-5c7ad0ed-2b79-4d59-92e2-b6d6e1af9934.png)

Reply:
How do you represent replies?

Reply:
> How do you represent replies?

Not sure if that's possible, I might fork/contribute and implement that

Reply:
> How do you represent replies?

Found how to do it. It's almost like a slash command but slightly different. 
```jsx
<DiscordMessage author="Example" avatar="red">
  <div slot="interactions">
    <DiscordInteraction author="Example" avatar="green">
      Replied message content
    </DiscordInteraction>
  </div>
  Message content
</DiscordMessage>
```